src/components/Challenges/CodeChallenge/CodeChallenge.tsx

```diff
@@ -46,6 +46,17 @@ export const CodeChallenge = ({
 
   useEffect(resetChallenge, [id]);
 
+  // passes test when control & enter keys are pressed together
+  useEffect(() => {
+    window.addEventListener('keydown', (e: KeyboardEvent) => {
+      e.preventDefault();
```

Comment:
I think that e.preventDefault is unnecessary here and may prevent other events that we need from being received. Correct me if I'm wrong